pandas/tseries/resample.py

```diff
@@ -85,8 +85,14 @@ def __init__(self, obj, groupby=None, axis=0, kind=None, **kwargs):
         self.exclusions = set()
         self.binner = None
         self.grouper = None
+        self.from_selection = False
 
         if self.groupby is not None:
+            # upsampling and PeriodIndex resampling do not work
+            # if resampling on a column or mi level
+            # this state used to catch and raise an error
+            self.from_selection = (self.groupby.key is not None or
```

Comment:
I would make this a private property, ``_from_selection``, instead of creating a new attribute directly.